worker/routes/paralegal.ts

```
@@ -0,0 +1,139 @@
+    const paralegalAgent = env.PARALEGAL_AGENT.get(doId);
+
+    // Route to appropriate DO endpoint
+    let doResponse: Response;
+    
+    switch (action.toLowerCase()) {
+      case 'advance':
+        if (request.method !== 'POST') {
+          throw HttpErrors.methodNotAllowed('POST method required for advance endpoint');
+        }
+        
+        const body = await request.json().catch(() => ({}));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)